test/unit/lualib/lualib.spec.ts

```diff
@@ -183,17 +159,15 @@ export class LuaLibArrayTests {
     @TestCase([1, 2, 3], "test", [5, 6], 7, ["test1", "test2"])
     @TestCase([1, 2, "test"], "test", ["test1", "test2"])
     @Test("array.concat")
-    public concat<T>(arr: T[], ...args: T[]): void {
+    public concat<T>(arr: T[], ...args: T[]): void
+    {
         const argStr = args.map(arg => JSON.stringify(arg)).join(",");
-        // Transpile
-        const lua = util.transpileString(
-            `let concatTestTable = ${JSON.stringify(arr)};
+
+        const result = util.transpileAndExecute(
+            `let concatTestTable: any[] = ${JSON.stringify(arr)};
```

Comment:
In the second-last test it will type concatTestTable as `Array<number | ConcatArray<number>>`, then assigning `"test"` to it results in invalid TypeScript.